src/matrix.jl

```diff
@@ -19,6 +19,12 @@ function initmatrix(filename::AbstractString)
     if !ok
         error("Invalid file $filename")
     end
+
+    # From lrs_close
+    ccall(:printf, Cint, (Ptr{Cchar},), "\n")
```

Comment:
Why is this needed ?

Reply:
Without it (i.e., with the current code on `master`), you have

```jl
julia> inem1 = LRSLib.HMatrix("simplex.ine");


julia> exit()
*Input taken from file simplex.ine(Terminal prompt)
```

I could have written

```jl
Libc.flush_cstdio()
println()
```

I don't know why one needs to flush `stdout` though.